### PR TITLE
Add fuzzy search

### PR DESCRIPTION
**Pull request by Epokhe**
I added fuzzy search using this package https://github.com/mokkabonna/inquirer-autocomplete-prompt

Thanks a lot for this tool, this was a very frustrating problem for me.

autocomplete

### CODE REVIEW

File: bin/grecent.js [hunk 1]
```diff
@@ -51,14 +55,20 @@ function listBranches([current, allBranches]) {
...
+
 function promptUser(branches) {
-  return inquirer.prompt({
-    type: 'list',
-    message: 'Choose branch:',
+  return inquirer.prompt([{
+    type: 'autocomplete',
     name: 'branch',
+    message: 'Choose branch:',
+    source: (answers, input) => Promise.resolve().then(() => filterBranches(input, branches)),
```
Comment (Epokhe) [inline bin/grecent.js:69]: https://github.com/mokkabonna/inquirer-autocomplete-prompt/issues/22

`source` requires a promise, I don't know if there is a better solution

Comment (jillesme) [reply bin/grecent.js:69]: Nice, I like it!

**Comment by jillesme** [comment]
Can you bump the package.json file according to semver too? I'll merge it after!

**Comment by Epokhe** [comment]
@jillesme New packages are at their latest versions available. Should I do something additionally? I searched a little but don't really understand what you mean fully 🙂

**Comment by jillesme** [comment]
No just package.json.version 1.0.0 to 1.1.0 (:

**Pull request merged**